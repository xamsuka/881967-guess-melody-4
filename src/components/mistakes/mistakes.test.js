import React from 'react';
import renderer from 'react-test-renderer';
import Mistakes from './mistakes.jsx';

test(`Mistakes component render`, () => {
  const three = renderer
    .create(<Mistakes countMistakes = {3} />).toJSON();

  expect(three).toMatchSnapshot(1);
});
