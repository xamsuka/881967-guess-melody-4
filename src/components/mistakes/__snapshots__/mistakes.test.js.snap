// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes component render 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
